pdfly/cli.py

```diff
@@ -205,6 +206,30 @@ def compress(
     pdfly.compress.main(pdf, output)
 
 
+@entry_point.command(name="uncompress", help=pdfly.uncompress.__doc__)  # type: ignore[misc]
+def uncompress(
+    pdf: Annotated[
+        Path,
+        typer.Argument(
+            exists=True,
+            file_okay=True,
+            dir_okay=False,
+            writable=False,
+            readable=True,
+            resolve_path=True,
```

Comment:
```suggestion
            dir_okay=False,
            exists=True,
            resolve_path=True,
```
Removing the parameters provided that are identical to the default values: https://github.com/fastapi/typer/blob/master/typer/params.py#L301